content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -816,6 +816,48 @@ You can optionally define a [custom unique constraint name](/concepts/components
 
 </Admonition>
 
+#### Composite type unique constraints
+
+In MongoDB in version `3.12.0` and later, you can define a unique constraint on a field `field` of a [composite type](#defining-composite-types) `compositeType` using the syntax `@@unique([compositeType.field])`. As with other fields, composite type fields can be used as part of a multi-column unique constraint.
```

Comment:
```suggestion
In version `3.12.0` and later, when using the MongoDB provider, you can define a unique constraint on a field `field` of a [composite type](#defining-composite-types) `compositeType` using the syntax `@@unique([compositeType.field])`. As with other fields, composite type fields can be used as part of a multi-column unique constraint.
```

Maybe something like that? To update below too

Reply:
@keerlu When saying `In MongoDB` it's suggesting that it's about MongoDB only (and not about MongoDB Prisma connector) I think

Reply:
So one could think MongoDB version `3.12.0` is a thing but in fact it's a Prisma version here

Reply:
Ah, ok, thanks. Reworded, hopefully that's better.